support-frontend/assets/components/directDebit/directDebitForm/directDebitForm.tsx

```diff
@@ -148,11 +148,12 @@ function LegalNotice(props: { countryGroupId: CountryGroupId }) {
 			</p>
 			<p>
 				<strong>Direct Debit</strong>
-				The Guardian, Unit 16, Coalfield Way, Ashby Park, Ashby-De-La-Zouch,
-				LE65 1JT United Kingdom
+				<address>
+					The Guardian, Mease Mill, Measham, Swadlincote, DE12 7DS
```

Comment:
Does `Westminster Industrial Estate` need to be added?

Reply:
@paul-daniel-dempsey thanks!
```suggestion
					The Guardian, Mease Mill, Westminster Industrial Estate, Measham, Swadlincote, DE12 7DS
```